src/backend/assets/js/session.js

```diff
@@ -1,20 +1,21 @@
 $(document).ready(function() {
+  // name is variable defined in the script in the session template file
+  var eventName = name;
```

Comment:
This part is of the earlier commit of improving the bookmark implementation done in the PR #1375. Please merge this PR after that has been merged. I will have to rebase it again to remove the old commit. Thanks :)